llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -2098,8 +2098,20 @@ unsigned SIInstrInfo::getNumWaitStates(const MachineInstr &MI) {
   }
 }
 
+Register SIInstrInfo::findImplicitExecSrc(const MachineInstr &MI) {
+  for (auto &Op : MI.implicit_operands()) {
+    if (Op.isDef())
+      continue;
+    Register OpReg = Op.getReg();
+    if (OpReg == AMDGPU::EXEC || OpReg == AMDGPU::EXEC_LO ||
+        OpReg == AMDGPU::SCC)
```

Comment:
The function name suggests it looks for implicit EXEC. Why is SCC here?
Also, it returns non-EXEC implicit Reg. The name is misleading.

Reply:
```findSetInactiveMask```?

Reply:
Looks good.